app/assets/javascripts/admin/addon/components/settings-editor.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="settings-editor">
+  <AceEditor
+    @mode={{"html"}}
+    @content={{this.editorContents}}
+  />
+  <br/>
+  <hr/>
+  {{#if this.errors}}
+    <div style="color:red">The following errors were found.</div>
```

Comment:
We have some alert styles in core we can use instead of the hardcoded colors here. The text should also use internationalization via the `i18n` helper, i.e. something like: 

```
{{i18n "admin.customize.theme.string_name"}}
```